config/sample-config.yaml

```diff
@@ -65,6 +65,76 @@ frontend:
     merge: false
     # The following configuration is taken from Network Observability Operator
     # see https://github.com/netobserv/network-observability-operator/blob/main/controllers/consoleplugin/config/static-frontend-config.yaml
+    panels: 
+    # Protocol
+    - Proto_Bytes
+    - sum_Proto_Bytes
+    - avg_Proto_Bytes
+    - Proto_Packets
+    - sum_Proto_Packets
+    - avg_Proto_Packets
+    - Proto_Flows
+    # DSCP
+    - Dscp_Bytes
+    - Dscp_Packets
+    - min_Dscp_TimeFlowRttNs
+    - max_Dscp_TimeFlowRttNs
+    - avg_Dscp_TimeFlowRttNs
+    - p90_Dscp_TimeFlowRttNs
+    - p99_Dscp_TimeFlowRttNs
+    - min_Dscp_DnsLatencyMs
+    - max_Dscp_DnsLatencyMs
+    - avg_Dscp_DnsLatencyMs
+    - p90_Dscp_DnsLatencyMs
+    - p99_Dscp_DnsLatencyMs
+    - Dscp_Flows
+    # Port numbers
+    - SrcPort_Bytes
+    - sum_SrcPort_Bytes
+    - SrcPort_Packets
+    - sum_SrcPort_Packets
+    - SrcPort_TimeFlowRttNs
+    - min_SrcPort_TimeFlowRttNs
+    - max_SrcPort_TimeFlowRttNs
+    - avg_SrcPort_TimeFlowRttNs
+    - p90_SrcPort_TimeFlowRttNs
+    - p99_SrcPort_TimeFlowRttNs
+    - DstPort_Bytes
+    - sum_DstPort_Bytes
+    - DstPort_Packets
+    - sum_DstPort_Packets
+    - min_DstPort_TimeFlowRttNs
+    - max_DstPort_TimeFlowRttNs
+    - avg_DstPort_TimeFlowRttNs
+    - p90_DstPort_TimeFlowRttNs
+    - p99_DstPort_TimeFlowRttNs
+    # Node Directions
+    - FlowDirection_Bytes
+    - FlowDirection_Packets
+    - FlowDirection_Flows
+    # TODO: implement a way to manage plurals for interfaces
+    # Interfaces Directions
+    #- IfDirections_Bytes
+    #- IfDirections_Packets
+    #- IfDirections_Flows
+    # Interfaces names
+    #- Interfaces_Bytes
+    #- sum_Interfaces_Bytes
+    #- avg_Interfaces_Bytes
+    #- Interfaces_Packets
+    #- sum_Interfaces_Packets
+    #- avg_Interfaces_Packets
+    #- Interfaces_Flows
```

Comment:
Since these fields went plural, this is not working anymore

Loki is supposed to be able to unwarp arrays but it will need some improvments on query side
https://github.com/grafana/loki/issues/4488#issuecomment-1128011841